libkbfs/folder_branch_ops.go

```diff
@@ -3967,6 +4101,8 @@ func (fbo *folderBranchOps) getCurrMDRevision(
 
 type applyMDUpdatesFunc func(context.Context, *lockState, []ImmutableRootMetadata) error
 
+var errNoUpdatesWhileDirty = errors.New("Ignoring MD updates while writes are dirty")
```

Comment:
Nit: can you use a new type instead? The problem with `errors.New` is that it records the stack at instantiation time, which might be confusing (probably still printed even if we call WithStack below)